backend/app.js

```diff
@@ -0,0 +1 @@
+"use strict";
```

Comment:
**thought:** This file is empty, maybe `server.js` could be moved here instead?

Reply:
Are you saying to bring server.js out to just backend/ ?

Reply:
Yeah, the contents of `server.js` can go here.